google_drive/templates/html/google_calendar_edit.html

```diff
@@ -0,0 +1,68 @@
+{% load i18n %}
```

Comment:
@mattdrayer not true!  in fact, I think we add aria too much here ;)  Only add when needed.  Here is a handy guide: http://www.w3.org/TR/aria-in-html/#rec

I did take a look at this as @martynjames said, and all that is required for making an ```<iframe>``` is to include a ```title``` attribute that describes what content will be embeded.  This allows the screen reader user to quickly identify, move focus to and decide if they want to consume the embedded content or skip over it.  BTW, this is really the only time I recommend using the ```title``` attribute.